### DISCUSSION TITLE
How to create a spawn on a specific position, city or base, enemy or civilian or both?

### DISCUSSION THREAD

**Discussion by LeQG-ARMA** [Vdauphin/HeartsAndMinds#1383] [Question and answer] [answered]
Hi Team. Already Respect for the work, it's just great so thank you. And if not more precisely, I cannot find in the documentation on how to create a city where there is not necessarily one to reference on the map? There is indeed creating a Faction, changing Map but not that, I seemed to have seen it however... A clarification please ;)

**Comment by LeQG-ARMA** [comment]
So far my search has landed me here:

`\=BTC=co@30_Hearts_and_Minds.xxx\core\fnc\city\create.sqf`

It is also necessary, I think, to set up a trigger on the card with an init looking like this:

`_city = [[xxx, xxx, xxx], "NameCityCapital", "BTC Capital", 500, true] call btc_city_fnc_create;`

But I'm absolutely not sure I'm right and fully understand what each piece of this line of code is.

**Comment by LeQG-ARMA** [comment]
The reason I'm looking to do this is that there are places on a map where there isn't a specific reference to a town or base, but there are buildings or a location that may denote of noteworthy interest, but there's no spawn , because mission scripts can't find them... Of course, the name doesn't exist. I hope I have been clear and explicit, I can obviously provide clarification if it is not understandable. Thanks in advance ;)

**Comment by Vdauphin** [comment, marked as answer]
Hello,
You can create city by following this template: https://github.com/Vdauphin/HeartsAndMinds/blob/master/%3DBTC%3Dco%4030_Hearts_and_Minds.Altis/define_mod.sqf#L1-L7

See tips here: http://vdauphin.github.io/HeartsAndMinds/Change-MAP-of-Hearts-and-Minds

Cheers

**Comment by LeQG-ARMA** [comment]
Many thanks, I thought I had seen something, but I couldn't remember. Thank you for bringing light to my prehistoric man cave ;)

**Comment by LeQG-ARMA** [comment]
**I'm adding clarification to the answer, if it can help with your TYPE setup. Each of them is in the form below**

![TYPE_Arma3](https://user-images.githubusercontent.com/65271259/175167464-9d497a80-d0d5-49e3-9498-eb1ee5824442.PNG)

**Arma 3**

    Airport (on Tanoa only. Tanoan airports have their own location type. On Altis and Stratis, airports are NameLocal)
    Area
    BorderCrossing
    CityCenter
    CivilDefense
    CulturalProperty
    DangerousForces
    Flag
    FlatArea
    FlatAreaCity
    FlatAreaCitySmall
    Hill
    HistoricalSite
    Invisible
    Mount
    Name
    NameCity
    NameCityCapital
    NameLocal (will return names like Airport)
    NameMarine
    NameVillage
    RockArea
    SafetyZone
    Strategic
    StrongpointArea
    VegetationBroadleaf
    VegetationFir
    VegetationPalm
    VegetationVineyard
    ViewPoint

[More details on the Wiki](https://community.bistudio.com/wiki/Location)
